rosidl_typesupport_opensplice_cpp/resource/msg__type_support.cpp.em

```diff
@@ -28,6 +28,7 @@
 #include "rosidl_typesupport_opensplice_cpp/identifier.hpp"
 #include "rosidl_typesupport_opensplice_cpp/message_type_support.h"
 #include "rosidl_typesupport_opensplice_cpp/message_type_support_decl.hpp"
+#include "rosidl_typesupport_opensplice_cpp/u__instanceHandle.h"
```

Comment:
is the semantic of this header different ? (to use the double underscore while every other header has a single one)